src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/JsonObject.cs

```diff
@@ -340,5 +420,357 @@ private static ICollection<object> PopulateHashTableFromJArray(JArray list, out
 
             return result;
         }
+
+        #endregion ConvertFromJson
+
+        #region ConvertToJson
+
+        /// <summary>
+        /// Convert an object to JSON string.
+        /// </summary>
+        public static string ConvertToJson(object objectToProcess, in ConvertToJsonContext context)
+        {
+            try
+            {
+                // Pre-process the object so that it serializes the same, except that properties whose
+                // values cannot be evaluated are treated as having the value null.
+                object preprocessedObject = ProcessValue(objectToProcess, currentDepth: 0, in context);
+                var jsonSettings = new JsonSerializerSettings
+                {
+                    TypeNameHandling = TypeNameHandling.None,
```

Comment:
Can you add a comment saying that this setting is required to be secure?

Reply:
Comment added: `// This TypeNameHandling setting is required to be secure.`